tests/functional/adapter/grants/test_invalid_grants.py

```diff
@@ -18,11 +18,10 @@
 from tests.utils.util import relation_from_name
 from dbt.tests.util import get_connection
 
-# Currently we return an HTTP error, but need to improve error handling
-# to get a better error message (https://github.com/dremio/dbt-dremio/issues/69)
+
 class TestInvalidGrantsDremio(BaseGrantsDremio, BaseInvalidGrants):
     def grantee_does_not_exist_error(self):
-        return "Can not fetch details for a job that is in [FAILED] state"
+        return "StatusRuntimeException: INTERNAL"
```

Comment:
These error messages do not correspond to their functions. Is it possible to align the error messages to the function names?

Reply:
Sounds good, thank you!